azurerm/internal/services/logic/resource_arm_logic_app_workflow.go

```diff
@@ -70,12 +91,23 @@ func resourceArmLogicAppWorkflow() *schema.Resource {
 				Default:  "1.0.0.0",
 			},
 
-			"tags": tags.Schema(),
-
 			"access_endpoint": {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"endpoint_configuration": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"workflow":  buildFlowEndpointsSchema(),
+						"connector": buildFlowEndpointsSchema(),
+					},
+				},
+			},
```

Comment:
(same here) these'll want to become top-level fields rather than nested within blocks

Reply:
Thank you Tom, make sense to flatten that. Also thanks to remind me we should use `TypeList` for computed collections for better user experience.